exercises/concept/train-driver/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+# Instructions
+
+Your friend is a train driver and has to drive cargo trains between cities.
+Although your friend isn't amazing with handling the computers and would like some help with it.
+Your friend would like your help organizing the train and correcting the mistakes in the data.
+
+
+```exercism/note
+To practice, use a for the rest or spread to solve each of the tasks below.
+```
+
+## 1. Convert the data to an array
```

Comment:
The header should fit the story, not represent the code, e.g. in this case "Create a list of all wagons".

Applies to all the headers.